docs/tools/aiven-console/howto/create-manage-teams.rst

```diff
@@ -54,3 +56,17 @@ To add projects and roles to a team:
 #. Click **Add project to team**.
 
 You can edit the permissions or delete the project from this team by clicking the more options menu for the project.
+
+.. _migrate_teams_to_groups:
+
+Migrate teams to groups
```

Comment:
Let's document fist the manual way. Which basically means:

1. Recreate all groups and add the users
2. Give the right permissions (roles) to the group on the project level
3. Adjust the SAML auto join settings (from Team to Group)
4. Remove the teams

Once we added this we will add Terraform next, the automated migration feature is on hold.

Reply:
Done in #2157.